### DISCUSSION TITLE
Vercel Build and Framework

### DISCUSSION THREAD

**Discussion by elle-concetta** [vercel/community#6860] [Help] [answered]
### Summary

My full stack app deploys, but I either get a "404 Not Found" or "Not Found" when deployment is completed. I'm not sure what to do. What's wrong with my build? And is there anything wrong with my code? Can someone please help?

<img width="1144" alt="Screenshot 2024-05-16 at 11 49 13 AM" src="https://github.com/vercel/community/assets/128157075/5f3ac6a5-7e9c-4bbf-8e39-8877f3326e85">

<img width="877" alt="Screenshot 2024-05-16 at 11 49 44 AM" src="https://github.com/vercel/community/assets/128157075/3ae5827c-377c-47d3-9885-dbe25a3aab52">

<img width="954" alt="Screenshot 2024-05-16 at 11 49 54 AM" src="https://github.com/vercel/community/assets/128157075/5d90b50e-aa6b-40b8-bb20-00eba9e06852">

<img width="941" alt="Screenshot 2024-05-16 at 11 50 08 AM" src="https://github.com/vercel/community/assets/128157075/d85ebc22-30b3-4c64-a5d1-6cb11107557b">

### Example

https://github.com/elle-concetta/FullStack-Wedding

### Steps to Reproduce

Here is the git repository my vercel project is connected to: https://github.com/elle-concetta/FullStack-Wedding

**Comment by swarnava** [comment, marked as answer]
https://github.com/elle-concetta/FullStack-Wedding/blob/e8cafef25bd6f8b61654eb6243bf55ebfdb7ef76/package.json#L6-L8

It looks like you are trying to run a server in Vercel which isn't possible since Vercel is a Serverless platform. You can follow this instruction to learn more about how to run express app on Vercel: https://vercel.com/guides/using-express-with-vercel
